crates/napi/src/bindgen_runtime/js_values/bigint.rs

```diff
@@ -226,20 +240,19 @@ impl From<u64> for BigInt {
 impl From<i128> for BigInt {
   fn from(val: i128) -> Self {
     let sign_bit = val < 0;
-    let words = (if sign_bit { -val } else { val }).to_ne_bytes();
+    let val = if sign_bit { -val } else { val };
     BigInt {
       sign_bit,
-      words: unsafe { std::slice::from_raw_parts(words.as_ptr() as *mut _, 2).to_vec() },
+      words: vec![val as _, (val >> 64) as _],
     }
   }
 }
 
 impl From<u128> for BigInt {
   fn from(val: u128) -> Self {
-    let words = val.to_ne_bytes();
     BigInt {
       sign_bit: false,
-      words: unsafe { std::slice::from_raw_parts(words.as_ptr() as *mut _, 2).to_vec() },
+      words: vec![val as _, (val >> 64) as _],
```

Comment:
This change does not change the asm output as well.
[before](https://play.rust-lang.org/?version=stable&mode=release&edition=2021&gist=b201f45dfa78ffcf9e0eb3fe569a8acb), [after](https://play.rust-lang.org/?version=stable&mode=release&edition=2021&gist=62fe5c6a94cea0e7b669d49f60e276b4)